src/apartment/mod.rs

```diff
@@ -34,12 +34,12 @@ impl Plugin for ApartmentPlugin {
                 interactable::interactable_system
                     .system()
                     .label("interactable_in_range"),
-            )
-            .add_system(
+            );
+            /* .add_system(
```

Comment:
Why keep this code?

Reply:
Because it wasn't mine and I didn't want to just delete other people's code but I guess we can move on